### PR TITLE
(WIP) Add headway widget

### PR DESCRIPTION
**Pull request by maxwofford**
Closes #97

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -31,6 +32,11 @@
           <%= image_tag 'logo.svg', name: 'header-logo', width: @home_size || 48, height: @home_size || 48, alt: 'Bank logo' %>
           <%= image_tag 'hank.png', name: 'alternative-logo', style: 'display: none;', width: @home_size || 48, height: @home_size || 48, alt: 'Hank logo' %>
         <% end %>
+        <% if signed_in? %>
```
Comment (lachlanjc) [inline app/views/layouts/application.html.erb]: Can we hide this if there aren't any updates? It feels really self-promotional if we have updates every other week & there's this huge button on every page

File: app/assets/stylesheets/components/_headway.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@import 'variables';
```
Comment (lachlanjc) [inline app/assets/stylesheets/components/_headway.scss]:
```suggestion
```

Variables are automatically imported by application.scss

File: app/assets/stylesheets/components/_headway.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  top: -5px;
+  left: 24px;
+}
+#HW_badge[data-count-unseen=""] {
+  opacity: 0 !important;
+}
+
+.headway {
+  color: map-get($palette, muted);
+}
```
Comment (lachlanjc) [inline app/assets/stylesheets/components/_headway.scss]:
```suggestion
}

```
Always have ending line break

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -62,5 +68,17 @@
       <% end %>
       }}();
     </script>
+    <script>
+      var HW_config = {
```
Comment (lachlanjc) [inline app/views/layouts/application.html.erb]:
```suggestion
      var HW_config = { selector: '.headway', account:  '7z8ovy', position: { y: 'bottom' }, translations: {} }
```
Would prefer this doesn't take up so much space in dev console

**Review by lachlanjc — APPROVED** [review]
A few minor code style things, looking good otherwise!

**Comment by lachlanjc** [comment]
@MaxWofford This is “WIP”—are more changes to it planned?

**Comment by maxwofford** [comment]
Nope, but I'll have a review session with @zachlatta before it's merged

**Comment by maxwofford** [comment]
(so I want to make sure it's clear no-one should merge this until I do)

**Comment by lachlanjc** [comment]
@MaxWofford I know we’re waiting on having changes in the log to ship, is this otherwise ready?

**Comment by maxwofford** [comment]
Yes, otherwise ready

Max Wofford
max@maxwofford.com
Engineering & Operations @ Hack Club ( https://hackclub.com )

**Pull request closed without merging**